das-dashboard/backend/controllers/query_controllers.py

```
@@ -0,0 +1,123 @@
+import json
+
+from fastapi import APIRouter, Query, WebSocket, WebSocketDisconnect
+from fastapi.logger import logger
```

Pylint (4.0.6)

[warning] Unused logger imported from fastapi.logger

(W0611)